modules/core/src/v2/coins/trx.ts

```diff
@@ -35,13 +41,25 @@ export interface TxInfo {
   from: string;
   txid: string;
 }
+export interface TronTransactionExplanation extends TransactionExplanation {
+  expiration: number;
+  timestamp: number;
+}
 
 export interface TransactionPrebuild extends BaseTransactionPrebuild {
   txHex: string;
   txInfo: TxInfo;
   feeInfo: TransactionFee;
 }
 
+export interface ExplainTransactionOptions {
+  txHex?: string; // txHex is poorly named here; it is just a wrapped JSON object
+  halfSigned?: {
+    txHex: string; // txHex is poorly named here; it is just a wrapped JSON object
+  };
+  fee: number;
+}
```

Comment:
I deleted the "txID" param that was here before. We no longer need to pass that in since txHex contains it.

Reply:
good one